src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SingleLineJavadocCheck.java

```diff
@@ -58,6 +58,45 @@
  * <pre>
  * &lt;module name=&quot;SingleLineJavadoc&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ *   public class Test {
+ *     /** &#64;return integer value *&#47; //violation
```

Comment:
`//violation` -> `// violation`
and other cases too, space between `//` and comment is import for readability. please update these changes at other places too.